virttest/qemu_vm.py

```diff
@@ -2080,11 +2084,9 @@ def add_disable_modern(devices, dev, dev_type):
         for balloon_device in params.objects("balloon"):
             params_balloon = params.object_params(balloon_device)
             balloon_devid = params_balloon.get("balloon_dev_devid")
-            balloon_bus = None
             use_ofmt = params_balloon.get("balloon_use_old_format",
                                           "no") == "yes"
-            if params_balloon.get("balloon_dev_add_bus") == "yes":
-                balloon_bus = pci_bus
+            balloon_bus = {"aobject": params_balloon.get("balloon_bus", "pci.0")}
```

Comment:
Right, according feature owner requirement we need to support add device to different bus. eg. virtio-blk-pci can be connect to pcie.0 and  pcie-switch downstream port or pci-bridge and so on, so I have above change.